qutebrowser/config/configdata.py

```diff
@@ -1677,4 +1677,6 @@ def data(readonly=False):
     (re.compile(r'^download-remove --all$'), r'download-clear'),
 
     (re.compile(r'^hint links fill "([^"]*)"$'), r'hint links fill \1'),
+
+    (re.compile(r'^yank-selected'), r'yank selection'),
```

Comment:
Can you please also add migrations for the previous keybindings with flags to `configdata.py`, and add tests for all migrations to `test_migrations` in `tests/unit/config/test_config.py`?